### DISCUSSION TITLE
How to configure fluentd?

### DISCUSSION THREAD

**Discussion by aagjalpankaj** [serversideup/docker-php#460] [General]
First of all thank you so much for this nice open-source project. It is really useful & time-saving for us.
My question:
`serversideup/docker-php` supports unified logging. How I can configure fluentd on top of `serversideup/docker-php` images? Do we have any example or documentation that I can refer?

**Comment by jaydrogers** [comment]
I haven't used Fluentd before, but all we're doing is sending everything out to stdout/stderr (which sends it to Docker).

You would need to configure your Docker engine to send to Fluentd, which I assume is compatible. Then you can monitor everything
